apm-sniffer/config/agent.config

```diff
@@ -92,6 +92,13 @@ logging.level=${SW_LOGGING_LEVEL:INFO}
 # The max recursive depth when checking the exception traced by the agent. Typically, we don't recommend setting this more than 10, which could cause a performance issue. Negative value and 0 would be ignored, which means all exceptions would make the span tagged in error status.
 # statuscheck.max_recursive_depth=${SW_STATUSCHECK_MAX_RECURSIVE_DEPTH:1}
 
+# Max element count in the correlation context.
+# correlation.element_max_number=${SW_CORRELATION_ELEMENT_MAX_NUMBER:3}
+# Max value length of each element.
+# correlation.value_max_length=${SW_CORRELATION_VALUE_MAX_LENGTH:128}
+# These tags of the span are derived from correlations with the same name.
+# correlation.injection_tags=${SW_CORRELATION_INJECTION_TAGS:}
+
```

Comment:
These are not widely used configurations, please don't add them in the `agent.config`.